build-local.rb

```diff
@@ -4,6 +4,8 @@
 require 'os'
 require 'tty-spinner'
 require 'yaml'
+require 'json'
+require 'pry'
```

Comment:
Are we able to drop `pry` now?

Reply:
I'll drop it when the PR moves out of draft :-D 

Having said that pry is mightly useful especially when trying new things or developing and I keep adding it in for Dev and removing it when the PR goes live, part of me would like to stop 🛑 having to do that cycle ;-)